onnx/defs/generator/defs.cc

```diff
@@ -91,12 +91,29 @@ ONNX_OPERATOR_SET_SCHEMA(
         .TypeConstraint("T1", {"tensor(int64)"}, "Constrain input types.")
         .TypeConstraint(
             "T2",
-            {"tensor(float16)",        "tensor(float)",      "tensor(double)",         "tensor(int8)",
-             "tensor(int16)",          "tensor(int32)",      "tensor(int64)",          "tensor(uint8)",
-             "tensor(uint16)",         "tensor(uint32)",     "tensor(uint64)",         "tensor(uint4)",
-             "tensor(int4)",           "tensor(bool)",       "tensor(bfloat16)",       "tensor(float8e4m3fn)",
-             "tensor(float8e4m3fnuz)", "tensor(float8e5m2)", "tensor(float8e5m2fnuz)", "tensor(float4e2m1)",
-             "tensor(float8e8m0)"},
+            {"tensor(float16)",
```

Comment:
You need to ensure the old op is preserved in the old.cc file, and bump this op to opset 25